coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -56,6 +51,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(SchemaUtilities.class);
   private static final String DALI_ROW_SCHEMA = "dali.row.schema";
 
+  private static final Set<String> USE_CALCITE_NULLABILITY_FUNCS =
+      Collections.unmodifiableSet(new HashSet<>(Arrays.asList("extract_union")));
+
```

Comment:
Is it better to create a registry class in Coral-Schema containing all the functions that need special handling and add detailed comments there?

Reply:
Here is not the best place to put the function list. I'm fine to put here for now if we only have this single function. However, please add comment what the list is and a TODO item to move it to a class if there are more those functions.

Reply:
Added a todo item.